README.md

```diff
@@ -74,8 +74,8 @@ options:
   -h, --help            show this help message and exit
   -i SRC_DIR_IMAGE [SRC_DIR_IMAGE ...], --src SRC_DIR_IMAGE [SRC_DIR_IMAGE ...]
                         Source directories, container images or binary files. Defaults to current directory.
-  -o SBOM_OUTPUT, --output-file SBOM_OUTPUT
```

Comment:
Let's keep --output-file to avoid a breaking change for users. stdout is also considered a file so is fine.